test/main_contracts/rate_oracles/compoundBorrowConfig.ts

```diff
@@ -0,0 +1,44 @@
+import { MockCToken, TestRateOracle } from "../../../typechain";
+import { toBn } from "../../helpers/toBn";
+import { deployments, ethers } from "hardhat";
+
+let cToken: MockCToken;
+const startingExchangeRate = 1;
+
+export const ConfigForGenericTests = {
+  configName: "CompoundBorrow",
+  startingExchangeRate,
+  oracleFixture: async () => {
+    // Use hardhat-deploy to deploy factory and mocks
+    await deployments.fixture(["Factory", "Mocks"]);
+    const token = await ethers.getContract("ERC20Mock");
+
+    // store the cToken and interest rate model and some rate per block info for use when setting rates
+    cToken = (await ethers.getContract("MockCToken")) as MockCToken;
+
+    // Set inital rate per block to 0
+    await cToken.setBorrowRatePerBlock(0);
```

Comment:
do we want to set this to a more realistic value?

Reply:
to make sure the interpolation works as expected

Reply:
the generic tests assume that the initial rate is 1 (which is set through the `borrowIndex`). tho we check if the interpolation works in the compound borrow tests - https://github.com/Voltz-Protocol/voltz-core/blob/ir/compound-borrow-rate-oracle/test/main_contracts/rate_oracles/compoundBorrowRateOracle.ts

Reply:
got it